<launch>
<arg name="flip" default="false"/>
<node pkg="ar_recog" type="ar_recog" name="ar_recog">
  <param name="image" value="/camera/image_raw" />
  <param name="aov" value="0.67"/>
  <param name="camera_para" value="$(find ar_recog)/bin/camera_para.dat"/>
  <param name="model_name" value="$(find fiducial_utils)/artoolkit_patterns/artag/object_data"/>
  <param name="min_confidence" value="0.6"/>
  <param name="flip" value="$(arg flip)"/>
</node>
</launch>
